### PR TITLE
Removed additional \

### PR DESCRIPTION
**Pull request by lunberg**
Previously it was written C:\\ so I have removed a slash so it reads C:\

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3fbeff046c/how-to/)

**Comment by WilliamLoy** [comment]
Article has been been SME reviewed.

File: content/cloud-servers/adding-disk-space-after-resizing-a-windows-server-2012-cloud-server.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ title: Add disk space after resizing a Windows Server 2012 Cloud Server
 type: article
 created_date: '2013-04-10'
 created_by: Rackspace Support
-last_modified_date: '2018-11-15'
-last_modified_by: Kate Dougherty
+last_modified_date: '201-01-23'
```
Comment (RackerKate) [inline content/cloud-servers/adding-disk-space-after-resizing-a-windows-server-2012-cloud-server.md]: 2019

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a71a4f8150/how-to/)

**Pull request merged**